cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -232,6 +233,7 @@ def setup_snapshot_env(test_vm, memory):
 @pytest.mark.parametrize("parent_vm", ["on_with_memory", "on_without_memory", "off"])
 @pytest.mark.uncollectif(lambda provider: not provider.one_of(VMwareProvider),
                          'Not VMware provider')
+@pytest.mark.uncollectif(BZ(1527617, forced_streams=['5.9']).blocks, 'BZ 1527617')
```

Comment:
Please use `pytest.mark.meta(blockers=[])` instead of uncollect, which will properly handle the skip reason in artifactor reports.